cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -0,0 +1,193 @@
+
+== AWS and Red Hat account and subscription linking
+
+. Click the orange *Continue to Red Hat* button to proceed with account linking:
++
+image::rosa-continue-rh-6.png[]
+
+. If you are not already in the same browser session, you will be asked to log into your Red Hat account:
++
+image::rosa-login-rh-account-7.png[]
++
+* You can also register for a new Red Hat account or reset your password on this page. 
+* Make sure to log into the Red Hat account that you plan to associate with the AWS account where you have activated {hcp-title} in previous steps.
+* Only a single AWS account that will be used for service billing can be associated with a Red Hat account. Typically an organizational AWS account that has other AWS accounts, such as developer accounts, linked would be the one that is to be billed, rather than individual AWS end user accounts.
+* Red Hat accounts belonging to the same Red Hat organization will be linked with the same AWS account. Therefore, you can manage who has access to creating {hcp-title} clusters on the Red Hat organization account level.
+
+. Complete the Red Hat account linking after reviewing the terms and conditions:
++
+[NOTE]
+====
+This step is available only if the logged-in Red Hat account, or the Red Hat organization managing the Red Hat account, was not linked to an AWS account before.
+====
++
+image::rosa-rh-account-connection-8.png[]
++
+Both the Red Hat and AWS account numbers are shown on this screen.
+
+. Click the *Connect accounts* button if you agree with the service terms.
++
+If this is the first time you are using the {hybrid-console}, you will be asked to agree with the general managed services terms and conditions before being able to create the first ROSA cluster:
++
+image::rosa-terms-conditions-9.png[]
++
+Additional terms that need to be reviewed and accepted will be shown after clicking the *View Terms and Conditions* button:
++
+image::rosa-terms-conditions-9-5.png[]
++
+Submit your agreement once you have reviewed any additional terms when prompted at this time.
+
+. The {hybrid-console-second} provides a confirmation that AWS prerequisites were completed and lists the first steps needed for cluster deployment:
++
+image::rosa-cluster-create-10.png[]
+
+. The following steps pertain to technical deployment of the cluster:
++
+image::rosa-deploy-11.png[]
++
+* It is possible that these steps will be performed on a different machine than where the service enablement and account linking were completed.
+* As mentioned previously, any Red Hat account belonging to the Red Hat organization that was linked with the AWS account that activated the ROSA service will have access to creating a cluster and will be able to select the billing AWS account that was linked under this Red Hat organization previously.
+
+The last section of this page shows cluster deployment options, either using the `rosa` CLI or though the web interface:
++
+image::rosa-cli-ui-12.png[]
```

Comment:
I think you need to remove the plus at 106 to fix this rendering break:

![image](https://github.com/openshift/openshift-docs/assets/16167833/013521f6-1c55-4592-92d6-c88e4089dd5e)